proto/client_policy.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package io.linkerd.proxy.client_policy;
+
+option go_package = "github.com/linkerd/linkerd2-proxy-api/go/client-policy";
+
+import "destination.proto";
+import "google/protobuf/duration.proto";
+import "meta.proto";
+import "http_route.proto";
+
+service ClientPolicies {
+  // Given a destination, return the ClientPolicy that is attached to that
+  // destination and send an update whenever it changes.
+  rpc GetClientPolicy(destination.GetDestination) returns(stream ClientPolicy) {}
```

Comment:
we've determined that the request type for this will need to include some kind of workload identifier for the pod that issued the lookup, so the controller can decide whose policybindings to use.

the `destination.GetDestination` message type currently includes the proxy's context token, which is set at inject time to
```json
{"ns":"$(_pod_ns)", "nodeName":"$(_pod_nodeName)"}
```
so, it includes the pod's namespace and node name, but it doesn't currently include the pod's actual name.

we'll eventually either want to send a request type that has the pod name, or change the context tokens to include the pod's name...